drivers/adc/adc_npcx.c

```diff
@@ -158,6 +174,34 @@ static inline uint32_t npcx_thrctl_reg(const struct device *dev,
 	return (config->base + config->threshold_reg_offset) + (ctl_no - 1) * 2;
 }
 
+#ifdef CONFIG_SOC_SERIES_NPCX4
+static inline uint32_t npcx_then_reg(const struct device *dev)
+{
+	const struct adc_npcx_config *config = dev->config;
+
+	return (config->base + config->threshold_reg_offset + 0x10);
```

Comment:
add a define for `0x10`

Reply:
Please see the [reply](https://github.com/zephyrproject-rtos/zephyr/pull/61996#discussion_r1310997013) below.